### ISSUE TITLE
csv import feature

### ISSUE THREAD

**Issue by Beastern** [towfiqi/serpbear#10]
There is already an option to export keywords as csv. It would be nice to have the ability to import them aswell. This also would make the entire workflow easily backupable and portable.

**Comment by towfiqi** [comment]
Good Idea but will have to think about it though since I am already planning on creating auto db backup feature.

**Comment by dreadedhamish** [comment]
The current export CSV only exports keywords and the latest ranking.
It would be great to be able to export an entire domain/project with all historical records, and likewise import.

**Comment by s3w47m88** [comment]
@Beastern until this feature is added, I recommend pasting your keywords list into ChatGPT and asking it to clean your list and provide it back in a bulleted list. I did this today and it took 30 seconds from start to finish. In my case, my list was not just keywords but ranking data, timestamps, and other unnecessary meta. GPT just stripped that all out, super easy.
